debian/rules

```diff
@@ -34,7 +34,11 @@ override_dh_install:
 	: # take care about it anyways
 	rm -rf $(DESTDIR)/var/run/ $(DESTDIR)/run/
 	: # Install monit configuration
-	install -d $(DESTDIR)/etc/monit/monitrc.d
+	if dpkg --compare-versions "$(dpkg-parsechangelog --show-field Version)" lt "1:5.15-2" 2> /dev/null; then
```

Comment:
this will parse our changelog... we should check which monit is available on the system, but that involves parsing apt-cache policy output etc... probably too much for the rules... 

so I just merged your original commit, added debian/changelog entry and pushed.

Reply:
Oh! Sorry.
I've got confused.